Add plugin: Inline Callouts

gapmiss opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/gapmiss/inline-callouts

## Release Checklist
- [X] I have tested the plugin on
  - [ ]  Windows
  - [X]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [X] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [X] `main.js`
  - [X] `manifest.json`
  - [X] `styles.css` _(optional)_
- [X] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [X] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [X] My README.md describes the plugin's purpose and provides clear usage instructions.
- [X] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [X] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [X] I have added a license in the LICENSE file.
- [X] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Why did you create a new plugin instead of updating the old one?

[Copyright (c) 2023 gapmiss](https://github.com/gapmiss/inline-callouts/blob/0fd72c2ccb3c7a9d91698a73f4e4f3d583a57327/LICENSE#L3)
Wrong year.

gapmiss commented:
> Why did you create a new plugin instead of updating the old one?

I changed to a different, simpler syntax and I didn't want to break user's already established badges. I understand I could have probably made it backwards compatible, but I assume users would rather just switch to a new-and-improved plugin.

Out of curiosity, how would you approach something like that (upgrading and making it backwards compatible)? Seems like a difficult task to accomplish and also maintain.

> [Copyright (c) 2023 gapmiss](https://github.com/gapmiss/inline-callouts/blob/0fd72c2ccb3c7a9d91698a73f4e4f3d583a57327/LICENSE#L3) Wrong year.

Fixed
